File: scode/readthedocs/prob1/Main.java
import java.util.*;
import java.io.*;
import java.lang.Math;
public class Main {
  public static void main(String args[]) throws IOException {
    Scanner sc= new Scanner(System.in);
    int test = sc.nextInt();
      while(test != 0){
        String a = sc.next();
        int no_of_operations = 0;
        for(int i=0; i<a.length()/2; i++){
          no_of_operations += Math.abs( a.charAt(a.length()-i-1) - a.charAt(i) );
        }
        System.out.println(no_of_operations);
        test--;
      }
  }
} 

File: scode/readthedocs/prob1/main.c
#include <stdio.h>
int main()
{
  int test;
  scanf("%d", &test);
  while(test--){
    char a[10000001];
    scanf("%s", a);
    int no_of_operations = 0;
    for(int i=0; i<strlen(a)/2; i++){
      no_of_operations += abs( a[strlen(a)-i-1] - a[i] );
    }
    printf("%d\n", no_of_operations);
  }
}